projects/packages/jitm/src/class-jitm.php

```diff
@@ -156,7 +170,10 @@ public function is_a8c_admin_page() {
 		return (
 			$current_screen
 			&& $current_screen->id
-			&& (bool) preg_match( '/jetpack|woo|shop|product/', $current_screen->id )
+			&& (bool) preg_match(
```

Comment:
Suggesting for now we gate this with `IS_WPCOM` to ensure the JITM code will not run on Simple until we're ready to deploy.